tools/packaging/kata-deploy/runtimeclasses/kata-runtimeClasses.yaml

```diff
@@ -80,21 +80,21 @@ scheduling:
 kind: RuntimeClass
 apiVersion: node.k8s.io/v1
 metadata:
-    name: kata-qemu-snp
-handler: kata-qemu-snp
+    name: kata-qemu-sev
+handler: kata-qemu-sev
 overhead:
     podFixed:
         memory: "2048Mi"
         cpu: "1.0"
 scheduling:
-  nodeSelector:
-    katacontainers.io/kata-runtime: "true"
+    nodeSelector:
+        katacontainers.io/kata-runtime: "true"
```

Comment:
Hey Choi, was this indentation update deliberate? (as vscode auto adds a bunch for me). We seem to have a mix of 2 space and 4 space indents on this file, so it's difficult to tell if it it the correct change or not, but I'm not against it as it seems to align with the rest of that yaml section

Reply:
Yeah, Otherwise `diff` does not return 0, which leads to the check failure. I had no choice but to do that change.

Reply:
I've updated the PR to preserve the indents for `kata-qemu-se`. Thanks for pointing it out.